templates/consulting/index.html

```diff
@@ -0,0 +1,515 @@
+              </p>
+              <p>
+                This Kubernetes deployment can run in any environment - from a single board device at the edge, to the biggest scale out cluster in a data center - and it includes the latest integrations with GPU-powered physical and virtual environments.
+              </p>
+              <p>We provide a full training course on Kubernetes deployment and operation.</p>
+            </div>
+          </div>
+        </div>
+
+        <div tabindex="0"
+             role="tabpanel"
+             id="data-platform-tab"
+             aria-labelledby="data-platform"
+             aria-hidden="false">
+          <div class="row--25-75-on-large">
+            <div class="col">
+              <h3 class="p-heading--5">Databases</h3>
+            </div>
+            <div class="col">
+              <p>
+                We provide enterprise support and deployment services for the most popular <a href="/data">open source databases</a>, such as MySQL, PostgreSQL, MongoDB, OpenSearch, and Valkey. We will design and deploy a production-grade open source database on any public, private or hybrid cloud.
+              </p>
+              <p>
+                We can deliver your database as a standalone deployment, or we can provide you with the functionality to offer self-service deployment to your end users.
+              </p>
+              <p>We also offer comprehensive training for these database applications.</p>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row--25-75-on-large">
+            <div class="col">
+              <h3 class="p-heading--5">Kafka</h3>
+            </div>
+            <div class="col">
+              <p>
+                Replace your enterprise edition Kafka with a <a href="/data/kafka">fully supported open source Kafka</a>. We will design and deploy a production-grade open source Kafka cluster and help you migrate all of your application messaging.
+              </p>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row--25-75-on-large">
+            <div class="col">
+              <h3 class="p-heading--5">AI/MLOps</h3>
+            </div>
+            <div class="col">
+              <p>
+                Turn your AI use case into reality with the following services:
+                <ul>
+                  <li>
+                    AI workshop – We run a workshop to gather your business and technical requirements and translate them to infrastructure and AI applications, which will lead to a design for your AI training and inference environments.
+                  </li>
+                  <li>
+                    MLOps design – We design your full stack AI environment from the ground up, including platform capacity planning, operating system, container orchestration, Kubeflow, MLflow, KServe, OpenSearch, Kafka, and applications for training and inference.
```

Comment:
Can you update the em-dash instances to `&mdash;`
```suggestion
                    MLOps design &mdash; We design your full stack AI environment from the ground up, including platform capacity planning, operating system, container orchestration, Kubeflow, MLflow, KServe, OpenSearch, Kafka, and applications for training and inference.
```